trend_micro_vision_one_xdr/README.md

```diff
@@ -0,0 +1,65 @@
+## Overview
+
+[Trend Micro Vision One XDR][1] collects and automatically correlates data across multiple security layers - email, endpoint, server, cloud workload, and network. This allows for faster detection of threats and improved investigation and response times through security analysis.
+
+This integration ingests the following logs:
+
+- Workbench Alerts
+- Observed Attack Techniques
+
+This integration seamlessly collects all the above listed logs, channeling them into Datadog for analysis. Leveraging the built-in logs pipeline, these logs are parsed and enriched, enabling effortless search and analysis. The integration provides insight into workbench alerts and observed attack techniques through the out-of-the-box dashboards.
+
+## Setup
+
+### Configuration
+
+#### Create API KEY from Trend Micro Vision One XDR
+
+1. On the Trend Vision One console, go to **Administration > API Keys** .
+2. Generate a new authentication token. Click **Add API key**. Specify the settings of the new API key.
+    - Name: A meaningful name that can help you identify the API key
+    - Role: The user role assigned to the key. Select **SIEM** from dropdown.
+    - Expiration time: The time the API key remains valid.
+    - Status: Whether the API key is enabled.
+    - Details: Extra information about the API key.
+3. Click **Add**.
+4. Copy and store the authentication token in a secure location.
+
+
+#### Trend Micro Vision One XDR DataDog Integration Configuration
+
+Configure the Datadog endpoint to forward Trend Micro Vision One XDR logs to Datadog.
+
+1. Navigate to `Trend Micro Vision One XDR`.
+2. Add your Trend Micro Vision One XDR credentials.
+
+| Trend Micro Vision One XDR Parameters | Description                                                  |
+| ------------------------------------- | ------------------------------------------------------------ |
+| Host Region                           | The Region of your Trend Micro Vision One XDR Console        |
+| API Key                               | The API Key of your Trend Micro Vision One XDR Console       |
+
+
+## Data Collected
+
+### Logs
+The Trend Micro Vision One XDR integration collects and forwards Workbench Alerts and Observed Attack Techniques logs to Datadog.
```

Comment:
Is there any setup required here?

Reply:
We have added the setup configurations steps above. Can you please let us know the exact requirement here?